core/domain/classroom_domain.py

```diff
@@ -17,14 +17,20 @@
 """Domain objects for Classroom."""
 
 from __future__ import annotations
+from typing import List
```

Comment:
```suggestion

from typing import List
```

Reply:
I tried to add these changes but I am getting lint check errors like this.
![v_3](https://user-images.githubusercontent.com/89520981/146015376-e6164010-dc23-428a-9b2e-ae56d87c6b0c.png)

Reply:
It seems to work, can you also just keep one line between `from __future__ import annotations` and `from typing import List`